lib/reek/cli/options.rb

```diff
@@ -9,11 +9,13 @@ module CLI
     #
     # Parses the command line
     #
+    # rubocop:disable ClassLength
```

Comment:
Can you move it to the [`FIXME` exclusion of `.rubocop.yml`](https://github.com/troessner/reek/blob/17c5fbce761727768c3d35c1bc12aa7abfa66c32/.rubocop.yml#L7-L10), please? We do want to fix it eventually. :)